docs/upgrade/capsule.md

```diff
@@ -24,7 +24,7 @@ Update the Foreman and Katello release packages:
 
 ```
   # yum update -y http://fedorapeople.org/groups/katello/releases/yum/{{ site.version }}/katello/el6/x86_64/katello-repos-latest.rpm
-  # yum update -y http://yum.theforeman.org/releases/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm
+  # yum update -y http://fedorapeople.org/releases/yum/foreman/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm
```

Comment:
I think you mean

https://fedorapeople.org/groups/katello/releases/yum/foreman/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm